README.md

```diff
@@ -251,8 +288,8 @@ Given `parseFooterTags: true`:
   "subject": "My first commit",
   "body": "Lorem ipsum dolor sit amet",
   "footer": {
-    "Closes": "#1",
-    "Foo": "bar"
+    "closes": "#1",
+    "foo": "bar"
```

Comment:
As per the previous example, shouldn't this now include `Closes`, `Foo`, `closes` & `foo` ?

Given that `lowerCaseFooterTags` presumably has no effect when `parseFooterTags` is false, would it have made sense to document `lowerCaseFooterTags` *after* `parseFooterTags` ?